src/microsoft/versions.json

```diff
@@ -133,26 +162,35 @@
           "GOARCH": "amd64",
           "GOOS": "linux"
         },
-        "sha256": "7b28ca61502d7034c32e0a03ecde15847db4ad323c2be88a14f3f6ffc065bff7",
+        "sha256": "5b07069f219f1d740e58ce3b562034e9d295f995bc5422ff05bf37b429263c4e",
         "supported": true,
-        "url": "https://dotnetbuildoutput.blob.core.windows.net/golang/microsoft/dev.boringcrypto.go1.18/20220414.3/go.20220414.3.linux-amd64.tar.gz"
+        "url": "https://dotnetbuildoutput.blob.core.windows.net/golang/microsoft/dev.boringcrypto.go1.18/20220421.1/go.20220421.1.linux-amd64.tar.gz"
+      },
+      "arm32v7": {
+        "env": {
+          "GOARCH": "arm",
+          "GOARM": "7",
+          "GOOS": "linux"
+        },
+        "sha256": "9c62001b8e1cfb2d1daf2c63108a2971f49152f498711903b477f89928204de9",
+        "url": "https://dotnetbuildoutput.blob.core.windows.net/golang/microsoft/dev.boringcrypto.go1.18/20220421.1/go.20220421.1.linux-armv6l.tar.gz"
```

Comment:
CI build fails: the upstream infra stops generating the Dockerfile when supported != true, but `dockerupdate` doesn't consider it while generating the manifest.json so the .NET Docker build infra tries to build Mariner armhf and fails to find the base tag (because it doesn't exist).

I've pushed a fix that manually removes these entries to see if it works, and now I'm working on a fix for `dockerupdate`.